packages/account/src/Sections/Security/Passwords/passwords-platform.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useRequest } from '@deriv/api';
+import { CFD_PLATFORMS, getPlatformSettings } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { localize } from '@deriv/translations';
+import FormSubHeader from 'Components/form-sub-header';
+import SentEmailModal from 'Components/sent-email-modal';
+import PlatformPartials from './platform-partials';
+
+type TPasswordsPlatformProps = {
+    has_dxtrade_accounts?: boolean;
+    has_mt5_accounts?: boolean;
+};
+
+/**
+ * Displays a change password instructions for MT5 and/or DXTrade.
+ * @name PasswordsPlatform
+ * @param [has_dxtrade_accounts=false] - Whether the user has DXTrade accounts.
+ * @param [has_mt5_accounts=false] - Whether the user has MT5 accounts.
+ * @returns {React.ReactNode}
+ */
+const PasswordsPlatform = observer(
+    ({ has_dxtrade_accounts = false, has_mt5_accounts = false }: TPasswordsPlatformProps) => {
+        const { mutate } = useRequest('verify_email');
+
+        const {
+            client: { email },
+        } = useStore();
+
+        const [identifier, setIdentifier] = React.useState('');
+        const [is_sent_email_modal_open, setIsSentEmailModalOpen] = React.useState(false);
+
+        const platform_name_dxtrade = getPlatformSettings('dxtrade').name;
+
+        const getPlatformTitle = () => {
+            let title = '';
+            if (has_mt5_accounts) {
+                title = localize('Deriv MT5 Password');
```

Comment:
```suggestion
                title = localize('Deriv MT5 Password');
```
Please use the <Localize /> component. The `localize()` function would be deprecated soon since it does not tie in well with the React lifecycle

Reply:
@yashim-deriv if you're referring to `Localize` component, it's a wrapper around `Trans` from `react-i18next` which says 
`It does ONLY interpolation. It does not rerender on language change or load any translations needed.`

Is there a workaround being added to this to rerender on language change?

ref: https://react.i18next.com/latest/trans-component

Reply:
The changes related to localize will be taken up while migrating FormSubHeader component